src/Microsoft.PowerShell.ConsoleHost/host/msh/Telemetry.cs

```diff
@@ -50,14 +73,18 @@ private static void SendTelemetry(string eventName, Dictionary<string,string>pay
             try
             {
                 // if the semaphore file exists, try to send telemetry
-                if (Utils.NativeFileExists(TelemetrySemaphoreFilePath))
+                var Enabled = Utils.NativeFileExists(TelemetrySemaphoreFilePath) && !GetEnvironmentVariableAsBool(TelemetryOptoutEnvVar, false);
```

Comment:
local vars should be camelCased

Reply:
oops, sorry about that one :-) I think I might have accidentally copied the style from somewhere else. I'll add another commit shortly to correct it.